CONTRIBUTING.md

```diff
@@ -1,79 +1,131 @@
+## Reporting a problem
 
-To contribute a fix, or add an update of the repository, do the following:
+To report an issue with the code, please submit it to our [issue tracker](https://github.com/neurodsp-tools/neurodsp/issues).
 
-1. Make a fork of the NeuroDSP repository
-2. Update the fork of the repository with any updates and/or additions to the project
-3. Check that any additions to the project follow the conventions (described below)
-4. Make a pull request from your fork to the `NeuroDSP` repository
-5. Address any feedback and/or recommendations from reviewers, until the contribution is ready to be merged
+In doing so, please try to include the following:
 
-## Scope
+1. A short, top-level summary of the issue (usually 1-2 sentences)
+2. A short, self-contained code snippet to reproduce the issue, ideally allowing a simple copy and paste to reproduce
+   - Please do your best to reduce the code snippet to the minimum required
+3. The actual outcome of the code snippet
+4. The expected outcome of the code snippet
 
-All contributions to `NeuroDSP` must be within the scope of the module.
+## Getting Involved
 
-The scope of `NeuroDSP` is digital signal processing approaches for neural data, that operate on neural time series. Within this scope, the goal of the module is to provide a collection of useful analyses approaches. These analyses should be agnostic to specific data modalities, and particularities of how the data is organized. Code and utilities that focused on data management are considered out of scope.
+We welcome all kinds of contributions to the project, including suggested features and help with documentation, maintenance, and updates.
 
-## Conventions
+If you have a new idea you would like to suggest or contribute, please do the following:
+
+1. Check if the idea is already being discussed on the
+   [issues](https://github.com/neurodsp-tools/neurodsp/issues) or
+   [development](https://github.com/neurodsp-tools/Development) page
+2. Check that your idea is within the [project scope](#project-scope)
+3. Open an [issue](https://github.com/neurodsp-tools/neurodsp/issues) describing
+   what you would like to see added / changed, and why
+4. Indicate in the issue if the idea is something you would be willing to help implement
+   - if so, project maintainers can give feedback to help make a plan for the contribution
+5. If you want to work on the contribution, follow the [contribution guidelines](#making-a-contribution) to do so
+
+If you are interested in getting involved and helping with the project, a great place to start is to visit the
+[issues](https://github.com/neurodsp-tools/neurodsp/issues) or
+[development](https://github.com/neurodsp-tools/Development) page
+and see if there is anything you would be interested in helping with. If so, join the conversation, and project developers can help get you started.
+
+## Project Scope
+
+All contributions to the module must be within it's scope.
```

Comment:
its

Reply:
ahhhhh, apostrophes! Good catch on the typo, thanks!